src/jwplayer/players/shared/JwPlayerWrapper.tsx

```diff
@@ -104,13 +104,19 @@ const JwPlayerWrapper: React.FC<JwPlayerWrapperProps> = ({
 				// Keep playing the ad when the user closed the mini player
 				if (viewable === 0 && pauseReason === 'external') {
 					playerInstance.play();
+				} else if (document.visibilityState === 'hidden' && pauseReason === undefined) {
+					// Keep the ad playing even when the user switches tabs
+					playerInstance.play();
```

Comment:
shouldn't we just remove this thing instead of making these changes? https://github.com/Wikia/ad-engine/blob/8712df48ac604fd89e887e9d4747b9acc201b72d/src/ad-products/video/jwplayer/helpers/jwplayer-helper.ts#L147C50-L147C73

Reply:
Nice finding. Yes, I think it's worth trying 👍